modules/ztp-filtering-ai-crs-using-siteconfig.adoc

```diff
@@ -0,0 +1,93 @@
+// Module included in the following assemblies:
+//
+// scalability_and_performance/ztp-deploying-disconnected.adoc
+
+:_module-type: PROCEDURE
+[id="ztp-filtering-ai-crs-using-siteconfig_{context}"]
+= Filtering custom resources using SiteConfig filters
+
+Using filters, you can easily customize `SiteConfig` custom resources (CRs) to include or exclude other CRs for use in the installation phase of the zero touch provisioning (ZTP) GitOps pipeline.
+
+The following example configures a custom single-node cluster `SiteConfig` CR that excludes the `siteconfig/extra-manifest/03-sctp-machine-config-worker.yaml` file from the install.
+
+.Prerequisites
+
+. Configure the hub cluster for generating the required installation and policy CRs.
+
+. Create a Git repository where you manage your custom site configuration data. The repository must be accessible from the hub cluster and be defined as a source repository for the Argo CD application.
+
+.Procedure
+
+. Create a set of custom manifest CRs that the ZTP pipeline uses to customize the cluster installs.
+
+. Open a shell prompt and change to your custom site directory. The following example illustrates the directory structure:
++
+[source,text]
+----
+.
+├── siteconfig
+│     ├── site1-sno-du.yaml
+│     └── extra-manifest
+│            └── 03-sctp-machine-config-worker.yaml
+└── user-custom-manifest
+      └── custom-sctp-machine-config-worker.yaml
+----
+
+. Edit the custom `site1-sno-du.yaml` file to set the `exclude` files, for example:
++
+[source,yaml]
+----
+apiVersion: ran.openshift.io/v1
+kind: SiteConfig
+metadata:
+  name: "site1-sno-du"
+  namespace: "site1-sno-du"
+spec:
+  baseDomain: "example.com"
+  pullSecretRef:
+    name: "assisted-deployment-pull-secret"
+  clusterImageSetNameRef: "openshift-4.10"
+  sshPublicKey: "<ssh_public_key>"
+  clusters:
+  - clusterName: "site1-sno-du"
+    networkType: "OVNKubernetes"
+    clusterLabels:
+      sites : "site1-sno-du"
+
+    ...
+
+    extraManifestPath:
+    extraManifests: <1>
+      filter:
+        exclude:
```

Comment:
Perhaps we should mention which CRs are exclusible? Seems it's easy to cause confusion unless user is familiar with how ztp works internally.

Reply:
"You can include or exclude any CRs from the folder specified in the `extraManifestPath` field."?

Reply:
The exclusible CRs are usually the ones built into the ztp site generator image under source-crs/extra-manifest/ dir. 
Although it can also exclude custom CRs customer added, but it is not a common use case, because it would have been easier to not add them at all. 
I'm not sure whats the best way to describe this. 

@vitus133 could you please suggest the wording?

Reply:
I would suggest "You can exclude any CRs from "source-crs/extra-manifest", which are otherwise included by default". Or something like that. To me it wouldn't make sense to add a CR to the extraManifestPath and then exclude it in the siteconfig

Reply:
I added this clarification to the procedure preamble